README.md

```diff
@@ -85,22 +89,20 @@ The AWS Athena driver is not officially deployed on a Maven repository, so you h
     <groupId>com.amazonaws.athena.jdbc</groupId>
     <artifactId>jdbcdriver</artifactId>
     <version>2.0.5</version>
-    <scope>system</scope>
-    <systemPath>${project.basedir}/AthenaJDBC42_2.0.5.jar</systemPath>
 </dependency>
 ```
 
-`Dockerfile` entry:
+Put the following line in your `Dockerfile` before the `RUN mvn install` command:
 
-```
-COPY AthenaJDBC42_2.0.5.jar /usr/src/app
+```bash
+RUN mvn install:install-file -Dfile=/usr/src/app/AthenaJDBC42_2.0.5.jar -DgroupId=com.amazonaws.athena.jdbc -DartifactId=jdbcdriver -Dversion=2.0.5 -Dpackaging=jar
```

Comment:
Not sure how this works, you'd have to put the downloaded Jar file into the `src/` folder? Perhaps clarify that in the instructions above?

Reply:
It didn't :)
We still need to copy the driver in the Dockerfile. I have updated the PR.